### ISSUE TITLE
Error: Undefined property: stdClass::$tld

### ISSUE THREAD

**Issue by Naoray** [tightenco/lambo#65]
It's basically same problem as described in #61 but the other way around. If I edit `lambo` to use `domain` instead of `tld` I don't get any errors.

Changing [these lines](https://github.com/tightenco/lambo/blob/master/lambo#L178-L181) to the following would do the fix to enable both `tld` and `domain` configs from valet:
```bash
if [[ -f ~/.config/valet/config.json ]]; then
    TLD=$(php -r "echo property_exists(json_decode(file_get_contents('$HOME/.config/valet/config.json')), 'tld') ? json_decode(file_get_contents('$HOME/.config/valet/config.json'))->tld : json_decode(file_get_contents('$HOME/.config/valet/config.json'))->domain;")
else
    TLD=$(php -r "echo property_exists(json_decode(file_get_contents('$HOME/.valet/config.json')), 'tld') ? json_decode(file_get_contents('$HOME/.valet/config.json'))->tld : json_decode(file_get_contents('$HOME/.valet/config.json'))->domain;")
fi
```

but it looks very messy... Maybe someone could improve this code? I am unfamiliar with bash script and couldn't figure out how to work with php vars inside the script...

**Comment by calebporzio** [comment]
Just to be sure, can you run `valet -V` and post your version?

Also sanity check by running `valet install` and see if that fixes it?

**Comment by mattstauffer** [comment]
@Naoray I just ran through it with @calebporzio who fixed this bug in #63... looks like it somehow got rolled back or overwritten, so I just pushed out a new Lambo version. Mind trying again with the updated Lambo?

**Comment by Naoray** [comment]
Thanks @calebporzio & @mattstauffer that fixes it!

Guess on the laravel-valet version `domain` was updated to `tld`, but on the linux-laravel-valet fork I use it didn't change...
